app/Models/Group.php

```diff
@@ -602,37 +603,44 @@ public function getTimezoneAttribute($value)
     }
 
     // TODO We've started to refactor into scopes, but this isn't complete yet.
-    public function scopeMembers() {
+    #[Scope]
+    protected function members() {
```

Comment:
Neat: https://laravel-news.com/local-model-scopes-in-laravel-with-the-scope-attribute